tests/impl.cpp

```diff
@@ -3450,7 +3450,19 @@ result_t test_mm_extract_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 
 result_t test_mm_insert_epi64(const SSE2NEONTestImpl &impl, uint32_t i)
 {
-    return TEST_UNIMPL;
+    const int64_t *_a = (const int64_t *) impl.mTestIntPointer1;
+    const int imm8 = 1;
```

Comment:
I think this should be a random value which would be mod by `2`.

Reply:
imm8 should take constant value at compiling time.
Using mod operation for a random number would cause error

Reply:
I will use fixed name temporarily

Reply:
Let's keep the original design.